student/src/main/java/tds/student/services/remote/RemoteOpportunityService.java

```diff
@@ -252,4 +289,22 @@ public void waitForSegment(final OpportunityInstance oppInstance, final int segm
   public void exitSegment(final OpportunityInstance oppInstance, final int segmentPosition) throws ReturnStatusException {
     legacyOpportunityService.exitSegment(oppInstance, segmentPosition);
   }
+  
+  private static TestConfig mapExamConfigurationToTestConfig(ExamConfiguration examConfiguration) {
+    Exam exam = examConfiguration.getExam();
+    TestConfig testConfig = new TestConfig();
+    testConfig.setStatus(OpportunityStatusExtensions.parseExamStatus(examConfiguration.getStatus()));
+    testConfig.setRestart(exam.getRestartsAndResumptions());
+    testConfig.setTestLength(examConfiguration.getTestLength());
+    testConfig.setStartPosition(examConfiguration.getStartPosition());
+    testConfig.setContentLoadTimeout(examConfiguration.getContentLoadTimeoutMinutes());
+    testConfig.setInterfaceTimeout(examConfiguration.getInterfaceTimeoutMinutes());
+    testConfig.setRequestInterfaceTimeout(examConfiguration.getRequestInterfaceTimeoutMinutes());
+    testConfig.setOppRestartMins(examConfiguration.getExamRestartWindowMinutes());
+    testConfig.setPrefetch(examConfiguration.getPrefetch());
+    testConfig.setScoreByTDS(false);
+    testConfig.setValidateCompleteness(examConfiguration.isValidateCompleteness());
```

Comment:
whoo boy.  Lotta properties.  Is this the only place we map an ExamConfiguration to a TestConfig or should we have a TestConfigBuilder that can copy an ExamConfiguration?